parts/k8s/kubernetesinstalls.sh

```diff
@@ -213,6 +213,7 @@ function extractHyperkube() {
     mv "/usr/local/bin/kubectl-${KUBERNETES_VERSION}" "/usr/local/bin/kubectl"
     chmod a+x /usr/local/bin/kubelet /usr/local/bin/kubectl
     rm -rf /usr/local/bin/kubelet-* /usr/local/bin/kubectl-*
+    rm -rf /home/rootfs-*
```

Comment:
Why is this here?

Reply:
because we want to remove k8s images at provisioning

Reply:
So it's just-in-case? The `mv` commands in `pullHyperkube()` should ensure that stuff doesn't accumulate in `/home/rootfs-*`, right?

Reply:
that won't be applied until a new VHD (we skip pullHyperkube at provisioning if image is already present)

Reply:
That will only be true after we build a new VHD image using the new version of `pullHyperkube` which uses `mv`, not `cp`. Currently with VHD 0.17.0, at provisioning all the `/home/rootfs-*` hyperkube files will be present. This is why I previously had comment 
"remove this with next VHD image".